app/src/main/java/com/speakeasy/watsonbarassistant/RecipeCollection.kt

```diff
@@ -0,0 +1,57 @@
+package com.speakeasy.watsonbarassistant
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.DividerItemDecoration
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import kotlinx.android.synthetic.main.activity_recipe_collection.*
+
+class RecipeCollection : AppCompatActivity() {
+
+    private var collectionName: String? = null
+    private var recipes: MutableList<Recipe> = mutableListOf()
+    private var recyclerView: RecyclerView? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_recipe_collection)
+
+        collectionName = intent.getStringExtra("Collection Name")
+        recipe_collection_title.text = collectionName
+        try {
+            @Suppress("UNCHECKED_CAST")

```

Comment:
Need to find a better way than suppressing the compiler error, but I don't think this will fail.